Makefile

```diff
@@ -53,14 +57,28 @@ build:
 # terraform {
 #  required_providers {
 #    aiven = {
-#      source  = "aiven/aiven"
+#      source  = "aiven-dev/aiven"
 #      version = "0.0.0+dev"
 #    }
 #  }
 #}
 build-dev: $(BUILD_DEV_DIR)
 	$(GO) build -o $(BUILD_DEV_DIR)/terraform-provider-aiven_v0.0.0+dev
 
+#################################################
+# Up
```

Comment:
What is the point of this block? I do not know how we can benefit of creating a folder and running TF apply as make commands

Reply:
it could be used for quick spin up of any Terraform project, and is useful for development